### PR TITLE
zookeeper_url split into two vars, Ease to change the downloada address

### PR DESCRIPTION
**Pull request by cifaz**

### CODE REVIEW

**Comment by ernestas-poskus** [comment]
@cifaz please add new variable to README.md as well and bump Zookeeper version to 3.4.12 because 3.1.11 is not available any more

**Comment by cifaz** [comment]
Add README.md i ok!  I upgrade the version to 3.4.12, and add vars zookeeper_base_url

File: defaults/main.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 zookeeper_version: 3.4.11
```
Comment (ernestas-poskus) [inline defaults/main.yml]: @cifaz you haved bumped in README.md but not in main.yml :)

**Review by ernestas-poskus — CHANGES_REQUESTED** [review]

File: defaults/main.yml (hunk 1 above)
Comment (cifaz) [reply defaults/main.yml]: And any else?

**Comment by ernestas-poskus** [comment]
There is an issue with java dependency https://travis-ci.org/AnsibleShipyard/ansible-zookeeper/jobs/372337448 I made a fix https://github.com/AnsibleShipyard/ansible-zookeeper/pull/85

**Comment by ernestas-poskus** [comment]
@cifaz please rebase I had to add this change https://github.com/AnsibleShipyard/ansible-zookeeper/pull/85 to require higher Ansible version

**Pull request closed without merging**